fractal.js

```diff
@@ -7,22 +7,26 @@ const path = require('path');
 const fractal = module.exports = require('@frctl/fractal').create();
 
 /* Set the title of the project */
-fractal.set('project.title', 'SFGOV Pattern Library');
+fractal.set('project.title', 'San Francisco Digital Services Design System');
```

Comment:
Maybe just "San Francisco Design System?" Ideally other agencies are using it...